components/script/dom/htmlinputelement.rs

```diff
@@ -292,12 +317,44 @@ impl HTMLInputElementMethods for HTMLInputElement {
 
     // https://html.spec.whatwg.org/multipage/#dom-input-value
     fn Value(&self) -> DOMString {
-        self.textinput.borrow().get_content()
+        match self.get_value_mode() {
+            ValueMode::Value => self.textinput.borrow().get_content(),
+            ValueMode::Default => {
+                self.upcast::<Element>()
+                              .get_attribute(&ns!(), &atom!("value"))
```

Comment:
What's the right indentation?  Should I run this chunk through rustfmt?

I updated rust-mode recently and everything is off.